util/configure

```diff
@@ -628,21 +628,12 @@ _END_
 
         {
             # check -msse4.2
-            my ($out, $cfile) = tempfile("resty-config-XXXXXX",
-                                         SUFFIX => '.c', TMPDIR => 1,
-                                         UNLINK => 1);
-
-            print $out "int main(void) { return 0; }";
```

Comment:
Can we actually use SSE 4.2 instructions in this test C program and actually runs the resulting binary to see if it returns 0 code?